code/game/objects/effects/countdown.dm

```diff
@@ -0,0 +1,76 @@
+/obj/effect/countdown

```

Comment:
I did not know this was a thing

Reply:
It's a thing in /tg/, aye.
It's in the PR text:
`Did a few small improvements, such as showing ghosts the timer on the nuke, and making it start anchored to avoid xeno pushing memes.`

Reply:
Oh i understand you doing it, i just didn't know text on screen like this was this easy. Means we could have more context for certain things instead of just icons.

For example when people shout medic, to just output medic / help /etc

Reply:
It doesn't look great, but yea, it's a possibility BYOND offers.